apps/mobile/src/libs/components/text/text.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react";
+import { Text as RNText, TextStyle } from "react-native";
+
```

Comment:
We have a Text import (needs to be changed to RNText) from “~/libs/components/components”;
And we can add a StyleProp import from “~/libs/types/types”

Reply:
I don't clearly understand your comment. I should import Text not from "react-native" from ~/libs/components/components?

Reply:
We already have `export { Text, View } from "react-native";` in ~/libs/components/components
So you can do `export { Text as RNText, View } from "react-native";` in ~/libs/components/components and `import { RNText }` in your component from ~/libs/components/components instead of importing from 'react-native'

Reply:
I believe we will remove an export from react-native from ~/libs/components/components, so I think it's ok to keep this
From ~/libs/components/components we will only export custom Text component

Reply:
Ah, ok, then leave the variant you had at the beginning:
`import { Text as RNText, TextStyle } from "react-native";`